packages/aws-cdk-lib/aws-rds/lib/instance-engine.ts

```diff
@@ -1131,7 +1344,10 @@ export class PostgresEngineVersion {
    * @deprecated PostgreSQL 11.15 is no longer supported by Amazon RDS.
    */
   public static readonly VER_11_15 = PostgresEngineVersion.of('11.15', '11', { s3Import: true, s3Export: true });
-  /** Version "11.16". */
+  /**
+   * Version "11.16"
+   * @deprecated PostgreSQL 11.16 is no longer supported by Amazon RDS.
```

Comment:
It seems like v11 is now deprecated ([docs](https://docs.aws.amazon.com/AmazonRDS/latest/PostgreSQLReleaseNotes/postgresql-release-calendar.html))

Reply:
Again, based on the SDK:

```sh
$ aws rds describe-db-engine-versions --engine postgres --region us-east-1 | jq -r '.DBEngineVersions[] | select(.EngineVersion | startswith("11.")) | .EngineVersion,.Status'
11.17
available
11.18
available
11.19
available
11.20
available
11.21
available
11.22
available
```